data-protocols/dsp/dsp-api-configuration/src/main/java/org/eclipse/edc/protocol/dsp/api/configuration/DspApiConfigurationExtension.java

```diff
@@ -37,25 +38,23 @@
 @Extension(value = DspApiConfigurationExtension.NAME)
 @Provides({ DspApiConfiguration.class })
 public class DspApiConfigurationExtension implements ServiceExtension {
-    
+
     public static final String NAME = "Dataspace Protocol API Configuration Extension";
-    
     public static final String CONTEXT_ALIAS = "protocol";
-    
     public static final String DEFAULT_DSP_CALLBACK_ADDRESS = "http://localhost:8282/api/v1/dsp";
+    @Setting(value = "Callback URL address fot the DSP")
```

Comment:
pls improve

Reply:
please add the `defaultValue` property as well